sw/device/sca/aes_serial.c

```diff
@@ -1,7 +1,6 @@
 // Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
-
```

Comment:
nit: I think we have this empty line in basically every c file. Would you mind bringing it back?